contracts/Arbitrum_SpokePool.sol

```diff
@@ -1,7 +1,4 @@
 // SPDX-License-Identifier: BUSL-1.1
-
-// Arbitrum only supports v0.8.19
-// See https://docs.arbitrum.io/for-devs/concepts/differences-between-arbitrum-ethereum/solidity-support#differences-from-solidity-on-ethereum
 pragma solidity ^0.8.19;
```

Comment:
I think we just don't _need_ to use 0.8.19 anymore: https://docs.arbitrum.io/build-decentralized-apps/arbitrum-vs-ethereum/solidity-support, so we can change `hardhat.config.ts` to use the same solc version as all our other contracts for the Arbitrum ones.

Reply:
Can we confirm this? If so, then yes we can change this. We should also check on these other contracts we cap at 8.19 [here](https://github.com/across-protocol/contracts/blob/d2b2b98dad18667b32cc022303ebc30b3acba5f7/hardhat.config.ts#L68)

Reply:
Sounds good. I updated the hardhat config as well here: https://github.com/across-protocol/contracts/pull/728/commits/12c4d532508329d9d7b12b22d33e467f28414532. Arbitrum's support for `PUSH0` was confirmed in the writeup for `N-13` as well. We should also be fine for Aleph Zero since they deployed ArbOS20, and the documentation says it was supported in an earlier version.

Reply:
So should we change this line to be ^0.8.0?

Reply:
Yeah. Done here https://github.com/across-protocol/contracts/pull/728/commits/8e4376964270bb50edfafa3d41becf02e9e81823